docs/release-notes/release-notes-0.19.0.md

```diff
@@ -249,6 +249,8 @@ The underlying functionality between those two options remain the same.
     - [Abstract autopilot access](https://github.com/lightningnetwork/lnd/pull/9480)
     - [Abstract invoicerpc server access](https://github.com/lightningnetwork/lnd/pull/9516)
     - [Refactor to hide DB transactions](https://github.com/lightningnetwork/lnd/pull/9513)
+    - Move the graph cache out of the graph CRUD layer:
```

Comment:
It just occurs to me that this feature is targeted for 0.20 or 0.19?

Reply:
depends on when we merge the base branch into master i guess! So let's leave this here for now and then if by the time we want to merge the base branch in, if 0.19 has shipped already, then i can move this to 0.20